src/it/scala/io/iohk/ethereum/txExecTest/util/DumpChainApp.scala

```diff
@@ -150,7 +150,7 @@ class BlockchainMock(genesisHash: ByteString) extends Blockchain {
       rootHash: NodeHash,
       position: BigInt,
       ethCompatibleStorage: Boolean
-  ): Option[(BigInt, Seq[MptNode])] = None
+  ): (BigInt, Seq[MptNode]) = (BigInt(0), Seq.empty)
```

Comment:
minor: 
The following tuple can be extracted as a trait, a class and an empty object, like: 
```
trait Proof { 
  value: BigInt,
  proof: Seq[ByteString]
}

class NonEmptyProof ...

object EmptyProof extends Proof { 
  override val value = BigInt(0)
  override val proof = Nil
}
```

Reply:
a proof for non-existing value is not necessarily empty, it can be a list of all nodes traversed while trying to find the value. So the only difference between empty and non-empty is the default value, but I like the idea of replacing a tuple with a class and having the default value for the `value` parameter then.